### PR TITLE
 [feature] Add RoutePoint for NeTEx France

### PR DESCRIPTION
**Pull request by woshilapin**

### CODE REVIEW

File: src/documentation/ntfs_to_netex_france_specs.md [hunk 1]
```diff
@@ -467,7 +467,58 @@ anticlockwise | anticlockwise
...
+- the `@order`
+
+For the rest of the identifier, use [id formatting](#id-of-objects).
+
+**(2) `@order` of the PointOnRoute**
+
+All the `stop_point` of the `route` are inserted with the following rules
+(`@order` is auto-incremented from `1`).
+
+The `vehicle_journeys` are processed in by alphanumerical order of their
```
Comment (ArnaudOggy) [inline src/documentation/ntfs_to_netex_france_specs.md]: alphanumerical to remove

Comment (woshilapin) [reply src/documentation/ntfs_to_netex_france_specs.md]: Done in ff10b51c119dd2c22fb21e441eaa4ea7088c0098.

File: src/netex_france/offer.rs [hunk 2]
```diff
@@ -33,17 +36,56 @@ type JourneyPattern = VehicleJourney;
...
+    route_points: BTreeMap<&'a str, Vec<Idx<StopPoint>>>,
+}
+
+fn calculate_route_points<'a>(model: &'a Model) -> BTreeMap<&'a str, Vec<Idx<StopPoint>>> {
+    model
+        .routes
+        .values()
+        .map(|route| {
+            // Unwrap is safe here since we are iterating on existing routes
+            let route_idx = model.routes.get_idx(&route.id).unwrap();
```
Comment (ArnaudOggy) [inline src/netex_france/offer.rs]:
```suggestion
        .iter()
        .map(|(route_idx, route)| {
```
Not better ?

Comment (woshilapin) [reply src/netex_france/offer.rs]: Much, much better. Thanks for this nice catch. Done in b1099530cfb087e12aa33bfad632afc13e68c760.

File: src/netex_france/route_points.rs [hunk 3]
```diff
@@ -0,0 +1,512 @@
...
+            let vehicle_journey_2 = vehicle_journey(
+                String::from("vehicle_journey_2"),
+                vec![
+                    stop_time_1.clone(),
+                    stop_time_2.clone(),
+                    stop_time_3.clone(),
+                ],
+            );
+            let vehicle_journeys = vec![vehicle_journey_1, vehicle_journey_2];
+            let route_points = build_route_points(vehicle_journeys.iter());
```
Comment (ArnaudOggy) [inline src/netex_france/route_points.rs]:
```suggestion
            let route_points = build_route_points(&vehicle_journeys);
```

Comment (woshilapin) [reply src/netex_france/route_points.rs]: Nice. I fixed it.

**Pull request merged**